tests/e2e/esm/index.test.js

```diff
@@ -544,7 +544,8 @@ describe('e2e esm tests', async () => {
   });
 
   describe('ANT', async () => {
-    const processId = 'YcxE5IbqZYK72H64ELoysxiJ-0wb36deYPv55wgl8xo';
+    // ANT v8 source
+    const processId = 'oQ4GNTed8cnNw-H5olq606gCFd5MbGSV4NTmfpIW4FI';
```

Comment:
nit: let's add a test for an older ant as well - can be in just a single `it` block

Reply:
[0489cb6](https://github.com/ar-io/ar-io-sdk/pull/259/commits/0489cb6924b68c19b8791cc47e0f7c1589be842f) - just a single getInfo call

Reply:
it should be validating the getRecords api